packages/nodes-base/nodes/Code/Code.node.ts

```diff
@@ -108,7 +108,7 @@ export class Code implements INodeType {
 				: 'javaScript';
 		const codeParameterName = language === 'python' ? 'pythonCode' : 'jsCode';
 
-		if (!runnersConfig.disabled && language === 'javaScript') {
+		if (!runnersConfig.disabled) {
```

Comment:
I think this will cause the Python Code node to stop working until we get the Python Task Runner going.

Reply:
Good catch! Not sure why I thought it would be ok to remove that 🤔 I'll fix it